src/extension.ts

```diff
@@ -200,6 +200,7 @@ export async function activate(context: VSCode.ExtensionContext) {
           omnisharpDirectory: Path.resolve(context.extensionPath, 'omnisharp'),
           csharpOssPath: Path.resolve(context.extensionPath, 'analyzers', 'sonarcsharp.jar'),
           csharpEnterprisePath: Path.resolve(context.extensionPath, 'analyzers', 'csharpenterprise.jar'),
+          eslintBridgeServerPath: Path.resolve(context.extensionPath, 'eslint-bridge')
```

Comment:
See my comment on SLLS: the `additionalAttributes` field is sent as-is to the telemetry, so I believe it needs to be renamed and cleaned-up.